includes/definitions/cambium.yaml

```diff
@@ -7,8 +7,9 @@ over:
 group: cambium
 discovery:
     - sysObjectId:
-        - .1.3.6.1.4.1.17713.21
+        - .1.3.6.1.4.1.17713
```

Comment:
So changing this will pick up:

`Motorola PTP 54400 Full, Version 54400-09-02`

Is this a Cambium device?

From our stats, the `'/^Cambium/'` would pick up Cambium PTP 58600 Lite PTP Wireless Ethernet Bridge, Version 58600-10-08

If you can add a test unit that would help.

Reply:
Sorry, don't follow, where is `Motorola PTP 54400 Full, Version 54400-09-02` picked up?
I couldn't find any Cambium/Motorola PTP except the 250 and the 650 in Librenms.

Cambium is the 'new' name for a division of Motorola Wireless, the bit that Google didn't buy. They rebadged some firmwares, but if a device was EOL, they didn't issue a rebadged firmware, they do still provide hardware support to some extent though, and I guess if they acknowledged a big enough security hole to be worth patching, it would probably come on a rebadged firmware.

Basically however, enterprises.17713 used to be Motorola, now it's Cambium.

Forgot about the unit test...

Reply:
.1.3.6.1.4.1.17713 is the Cambium backhaul wireless OID, 
.1.3.6.1.4.1.17713.21 is one particular hardware variant.

Reply:
Ah, I see `Cambium PTP 58600 Lite PTP Wireless Ethernet Bridge, Version 58600-10-08` now.
Odd, there doesn't seem to be an OS to match that test...

Reply:
So... this is turning stream of conciousness on my part...
.1.3.6.1.4.1.17713.21 is an ePMP device. The cambium.snmprec sysDescr uses the string, `WaveDirect` which is the name of the company who did the original discovery.
They added discovery for the various pieces of hardware they were using, in a rather haphazard manner. the Motorola/Cambium MIBS seem to be well written, they are broken down by hardware variant, then they follow a consistent structure wherever possible so I think things can be much better.

I'd like to have time to rewrite the entire Cambium / Canopy / Motorola discovery and polling to reflect the new wireless paradigm and achieve some consistency across the product line but I have two problems:
1. I am quite unclear about the structure and concepts of the new wireless paradigm;
2. I have about 10 other responsibilities beyond getting LibreNMS integrated in house...

Is there a reasonably concise guide to, or description of the wireless paradigm?
Can I simply tidy this up for now to get things working and open a new issue / PR to re-factor Cambium family discovery later?

Reply:
So `Motorola PTP 54400 Full, Version 54400-09-02` is from the stats we get from users.

I'll leave it up to you (or someone who knows this vendor better) to decide the best way forward. I was merely pointing out that the sysObjectID you've changed to will also pick up that motorola device so if that's not expected then you can't use that sysObjectID on it's own.

The wireless support has been revamped, some old devices which were shoe horned in where they could be haven't been updated yet. If you want to add more / extend the wireless stats for an OS then follow the new docs on that and ignore what was there before.

Reply:
Yes, all of these devices are no longer Motorola they are Cambium.

However, it makes me sad that when I finally get around to converting these to the new code, this is just one more thing to convert.